tests/TestHelpers/GraphHelper.php

```diff
@@ -97,6 +97,21 @@ public static function getFullUrl(string $baseUrl, string $path): string {
 		return $fullUrl;
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $path
+	 *
+	 * @return string
+	 */
+	public static function getBetaFullUrl(string $baseUrl, string $path): string {
+		$fullUrl = $baseUrl;
+		if (\substr($fullUrl, -1) !== '/') {
+			$fullUrl .= '/';
+		}
+		$fullUrl .= 'graph/v1beta1/' . $path;
+		return $fullUrl;
```

Comment:
is this to add the `/` at the end?
```suggestion
		$baseUrl = rtrim($baseUrl, "/");
		return $baseUrl . '/graph/v1beta1/' . $path;
```

Reply:
Just copy paste from line 91-98 https://github.com/owncloud/ocis/pull/7946/files#diff-c0b08c052523f739b28c50d0bd9f00c64067d8d56fd984ff03b06e34a8f6cdb8R91-R98

Reply:
Ahh, you can use this simple one if feel like